core/domain/collection_services.py

```diff
@@ -1226,13 +1226,14 @@ def get_acquired_skill_ids_of_user(user_id, collection_id):
 
     Returns:
         list(str). A list of skill ids acquired by the user.
+
+    Raises:
+        Exception: Collection with given ID does not exist.
     """
     completed_exploration_ids = get_completed_exploration_ids(
         user_id, collection_id)
     collection = get_collection_by_id(collection_id)
-    acquired_skills_ids = set()
-    if collection is not None:
-        acquired_skills_ids = (
-            collection.get_acquired_skill_ids_from_exploration_ids(
-                completed_exploration_ids))
+    acquired_skills_ids = (
```

Comment:
Nit: To be consistent, call this acquired_skill_ids ("skill" not "skills"). The function name is get_acquired_skill_ids_* so this should follow the same pattern.